tools/update-rules-list.ts

```diff
@@ -1,21 +1,26 @@
 import rules from './utils/rules'
 
-import { configBadges, emojiKey, writeRulesListInReadme, updateRulesDocs } from './utils/docs'
+import { emojiKey, writeRulesListInReadme, updateRulesDocs } from './utils/docs'
 
 /*
 This script updates the rules table in `README.md`from rule's meta data.
 */
 
+export type TRulesList = readonly [
+  ruleName: string,
+  ruleLink: string,
+  docsDescription: string,
+  fixable: string,
+  categories: string
+]
+export type TRuleListWithoutName = TRulesList extends readonly [string, ...infer TRulesWithoutName]
+  ? TRulesWithoutName
+  : never
+
 const createRuleLink = (ruleName: string) =>
   `[\`storybook/${ruleName}\`](./docs/rules/${ruleName}.md)`
 
-const generateConfigBadges = (recommendedConfig: any) =>
```

Comment:
Why was this deleted?

Reply:
As far as I remember, it was not used at all.

Reply:
Yup you're right, the idea was to be a draft of something I was planning, but so far it wasn't done so it better off be deleted. Thanks!